paddle/fluid/compiler/piano/backends/note_visitor_base.h

```diff
@@ -0,0 +1,71 @@
+// Copyright (c) 2021 PaddlePaddle Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+#pragma once
+
+namespace paddle {
+namespace piano {
+
+template <typename InstructionPtr>
+class NoteVisitorBase {
+ public:
+  virtual ~NoteVisitorBase() {}
+
+  // Scalar op
+  virtual void VisitConstant(InstructionPtr) = 0;
+
+  //
```

Comment:
Add this `// ops can be replaced by library` comment.

Reply:
done